potimeb/#Champion_of_Korascian.lua

```diff
@@ -0,0 +1,17 @@
+--Champion_of_Korascian (223099)
+--Phase 2 
+--potimeb
+
+function event_death_complete(e)
+	--signal phase_two_controller
+	eq.signal(223242,1);
+end
+
+function event_signal(e)
+	--signal from A Windshapen Warlord
+	if e.signal == 1 then
+		e.self:SetSpecialAbility(25, 0);
+		e.self:SetSpecialAbility(24, 0);
+		e.self:SetBodyType(1, true);
```

Comment:
Should use `SpecialAbility.immune_aggro`, `SpecialAbility.immune_aggro_on` and `BT.Humanoid` here instead (I know we don't do this perfectly everywhere, but we should!)